dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM spectreteam/python_msi:v4.0.4
+
+COPY . /app
+
+WORKDIR /app
+
+RUN apt update &&\
+  apt install --yes git &&\
+  pip install -r requirements.txt &&\
+  apt remove --yes git &&\
+  apt-get autoremove --yes
```

Comment:
Use `apt-get -qq <action>` instead of `apt <action> --yes` for the following reasons:
- for now, `apt` is a user-facing utility, without stable interface guarantee, unlike `apt-get`; hence the warning in build log
- `-qq` suppresses packet download info and implies `--yes`; as a result, only packet preparation and unpacking info will be shown, limiting clutter in the logs

There is also one step missing. You should delete downloaded package indices (generated by `apt-get update`) after installation.

```sh
rm -rf /var/lib/apt/lists/*
```

Reply:
That was the missing element! Thanks!